docs/developer/contributing.md

```diff
@@ -85,8 +85,10 @@ The Makefile provides several targets:
 ### Compile on Linux
 Compiling Grafana Agent on Linux requires a couple of extra dependencies:
 
-* [systemd headers](https://github.com/grafana/agent/blob/main/cmd/agent/Dockerfile#L8-L9) for Promtail
-* [bcc tools](https://github.com/grafana/agent/blob/main/cmd/agent/Dockerfile#L12-L13) for the eBPF integration on AMD64 systems
+* [systemd headers](https://packages.debian.org/sid/libsystemd-dev) for Promtail
+   * Can be installed on Debian-based distros with: ```sudo apt-get install libsystemd-dev```
+* [bcc tools](https://packages.debian.org/sid/libbpfcc-dev) for the eBPF integration on AMD64 systems
+   * Can be installed on Debian-based distros with: ```sudo apt-get install libbpfcc-dev```
```

Comment:
Can we spell out distros in full here? 

```suggestion
* [systemd headers](https://packages.debian.org/sid/libsystemd-dev) for Promtail
   * Can be installed on Debian-based distributions with: ```sudo apt-get install libsystemd-dev```
* [bcc tools](https://packages.debian.org/sid/libbpfcc-dev) for the eBPF integration on AMD64 systems
   * Can be installed on Debian-based distributions with: ```sudo apt-get install libbpfcc-dev```
```